src/modules/server/api/test/tldraw-document.api.spec.ts

```diff
@@ -1,11 +1,20 @@
-jest.mock('@y/redis');
+jest.mock('@y/redis', () => {
+	return {
+		registerYWebsocketServer: jest.fn(),
+		Api: jest.fn().mockImplementation(() => {
+			return {
+				prototype: jest.fn(),
+			};
+		}),
+	};
+});
 
+import { createMock } from '@golevelup/ts-jest';
 import { INestApplication } from '@nestjs/common';
 import { Test } from '@nestjs/testing';
-import { StorageService } from '../../../../infra/storage/storage.service.js';
-import { RedisService } from '../../../../infra/redis/redis.service.js';
 import { App } from 'uws';
-import { createMock } from '@golevelup/ts-jest';
+import { RedisService } from '../../../../infra/redis/redis.service.js';
```

Comment:
Der Pfad ist unglücklich. In dem Repo auch mit @infra zu arbeiten ist denke ich sinnvoller.

Reply:
Ja es soll auch so sein, leider wegen ESM Module ist das nicht so einfach wie im server. bzw noch nicht nachgeforscht